### DISCUSSION TITLE
Parallel strips and NeoTiles

### DISCUSSION THREAD

**Discussion by Johboh** [Makuna/NeoPixelBus#694] [General]
Hi!

I have a a led panel with 7x5 tiles, where each tile is 8x8 LEDs. Each column in this panel is connected to a separate pin on the ESP32-S2. Currently I have written my own mapping logic from X,Y coordinates to the correct strip and pixel.
However, I would like to use `NeoTiles` (or similar) instead and not need to switch between strips. Is there support for multiple strips together with `NeoTiles`? I've tried to search for example of this, but couldn't find any. If its out there, please point me in the right direction.

Regards,
Johan

**Comment by Makuna** [comment]
There isn't a specific solution for this yet.  On the surface it is a simple mapping, but often due to a specific design goals of sketch authors require different solutions causing a single model to not work well for everyone.

Issue https://github.com/Makuna/NeoPixelBus/issues/160 is tracking this general concept.  

If you would, you can add your "layout" and mapping to the issue as a way to capture models that need to be supported.

**Reply by Johboh** [reply]
Thanks, I will do so.

//Johan
